### ISSUE TITLE
<class 'libqtile.core.manager.Qtile'> has no attribute current_window

### ISSUE THREAD

**Issue by BTrey** [qtile/qtile#4783]
### Issue description

Upon starting, I see the below errors in the qtile.log. It does not appear to affect operation. Qtile still loads and functions normally. The errors are not seen again after start-up. (Below logs are from a manual restart but I get the error in the logs if I reboot and qtile loads normally.)

### Version

0.25.1.dev0+g005da458.d20240413

### Backend

X11 (default)

### Logs

```text
2024-04-28 09:08:06,314 WARNING libqtile lifecycle.py:_atexit():L33 Restarting Qtile with os.execv(...)
2024-04-28 09:08:06,795 ERROR libqtile core.py:_xpoll():L372 Got an exception in poll loop
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 347, in _xpoll
    self.handle_event(event)
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 314, in handle_event
    ret = target(event)
          ^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 781, in handle_DestroyNotify
    if self.qtile.current_window is None:
       ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 281, in __getattr__
    raise AttributeError(f"{self.__class__} has no attribute {name}")
AttributeError: <class 'libqtile.core.manager.Qtile'> has no attribute current_window
2024-04-28 09:08:06,799 ERROR libqtile core.py:_xpoll():L372 Got an exception in poll loop
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 347, in _xpoll
    self.handle_event(event)
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 314, in handle_event
    ret = target(event)
          ^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 781, in handle_DestroyNotify
    if self.qtile.current_window is None:
       ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 281, in __getattr__
    raise AttributeError(f"{self.__class__} has no attribute {name}")
AttributeError: <class 'libqtile.core.manager.Qtile'> has no attribute current_window
2024-04-28 09:08:06,802 ERROR libqtile core.py:_xpoll():L372 Got an exception in poll loop
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 347, in _xpoll
    self.handle_event(event)
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 314, in handle_event
    ret = target(event)
          ^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 781, in handle_DestroyNotify
    if self.qtile.current_window is None:
       ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 281, in __getattr__
    raise AttributeError(f"{self.__class__} has no attribute {name}")
AttributeError: <class 'libqtile.core.manager.Qtile'> has no attribute current_window
2024-04-28 09:08:06,806 ERROR libqtile core.py:_xpoll():L372 Got an exception in poll loop
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 347, in _xpoll
    self.handle_event(event)
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 314, in handle_event
    ret = target(event)
          ^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 781, in handle_DestroyNotify
    if self.qtile.current_window is None:
       ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 281, in __getattr__
    raise AttributeError(f"{self.__class__} has no attribute {name}")
AttributeError: <class 'libqtile.core.manager.Qtile'> has no attribute current_window
2024-04-28 09:08:06,809 ERROR libqtile core.py:_xpoll():L372 Got an exception in poll loop
Traceback (most recent call last):
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 347, in _xpoll
    self.handle_event(event)
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 314, in handle_event
    ret = target(event)
          ^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/backend/x11/core.py", line 781, in handle_DestroyNotify
    if self.qtile.current_window is None:
       ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/libqtile/command/base.py", line 281, in __getattr__
    raise AttributeError(f"{self.__class__} has no attribute {name}")
AttributeError: <class 'libqtile.core.manager.Qtile'> has no attribute current_window
```

### Required

- [X] I have searched past issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by elParaguayo** [comment]
I think this was fixed by #4757 so you would need to be on latest git version to fix this.

**Comment by BTrey** [comment]
Thanks. Not sure how my search-fu failed me in looking for the error, but I didn't find that bug at all. Will close.
